### ISSUE TITLE
vg xg doesn't work on whole genome graphs

### ISSUE THREAD

**Issue by cmarkello** [vgteam/vg#2242]
@adamnovak 
When running `vg xg` on a whole genome graph XG index to convert it to a VG graph, it fails due to exceeding a protobuf size limit of 2GB.

I've lost the error message, but I used the VG container `quay.io/vgteam/vg:v1.15.0-91-g5bda1fa5-t299-run` and the XG index file can be obtained from this URL `https://storage.googleapis.com/cmarkell-vg-wdl-dev/test_sv_map_calling/HGSVC.xg`

**Comment by ekg** [comment]
We should have it write GFA.

Everywhere we are using protobuf for output, we should replace it with at
least a text stream.

**Comment by adamnovak** [comment]
If we start parsing GFA in every subcommand, we have to potentially deal with GFA link overlaps in every subcommand. Are we going to bail out and tell people we can only deal with a restricted subset of GFA, or are we going to buffer the whole input and resolve the overlaps?

**Comment by ekg** [comment]
Yeah, we only deal with a restricted subset. GFA with no overlaps. I'd just
bail if encountering anything other than that.

The benefits of protobuf are seeming very slim to me. It's difficult to
work with the chunked model we've established. It requires graph traversal
logic to produce.

That said, you've done some cool stuff to make indexable .vg files.

**Comment by faithokamoto** [comment]
`vg xg` no longer exists
